### ISSUE TITLE
Implement cloudmesh Shell commands in windows

### ISSUE THREAD

**Issue by laszewsk** [cloudmesh/cloudmesh-catalog#9]
Before starting the task, see if it is not already done, be always up to date with a git pull. Implement for windows but do a pull request for each of the commands so it is easy to review. Please note that other community members could also implement this. The task was added as so many of you have windows machines and this will make it possible to do things portabal among different platforms

https://github.com/cloudmesh/cloudmesh-common/blob/main/cloudmesh/common/Shell.py

- [ ] cloudmesh.common.Shell.cat
- [ ] cloudmesh.common.Shell.head
- [ ] cloudmesh.common.Shell.tail
- [ ] cloudmesh.common.Shell.fgrep
- [ ] cloudmesh.common.Shell.grep
- [ ] cloudmesh.common.Shell.git
   git: if gitbash is installed or if its in the path

- [ ] cloudmesh.common.Shell.terminal
  add for windows cmd and powershell as an option specified with kind
